packages/std-client/package.json

```diff
@@ -9,20 +9,21 @@
   },
   "license": "MIT",
   "type": "module",
-  "main": "src/index.ts",
-  "source": "src/index.ts",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "src/index.ts",
   "scripts": {
     "build": "pnpm run build:mud && pnpm run build:js",
-    "build:js": "rollup -c rollup.config.js",
+    "build:js": "tsup",
     "build:mud": "tsx ./scripts/tsgen.ts",
     "clean": "pnpm run clean:mud && pnpm run clean:js",
     "clean:js": "rimraf dist",
     "clean:mud": "rimraf src/mud-definitions",
+    "dev": "tsup --watch",
     "docs": "rimraf API && typedoc src && find API -type f -name '*.md' -exec sed -E -i \"\" \"s/(#.*)(<.*>)/\\1/\" {} \\;",
-    "prepack": "mv package.json package.json.bak && jq \".main = \\\"dist/index.js\\\"\" package.json.bak > package.json ",
-    "postpack": "mv package.json.bak package.json || echo 'no package.json.bak'",
     "release": "npm publish --access=public",
-    "test": "pnpm run build:mud && vitest typecheck --run && echo 'todo: add tests'"
+    "test": "pnpm run build:mud && tsc --noEmit && vitest typecheck --run"
```

Comment:
having both tsc and vitest's typecheck seems redundant
I usually use tsc when there're no actual tests for vitest to run properly

Reply:
ahh does vitest also do a full typecheck?

Reply:
yeah it should be just strictly better than tsc for this purpose, it just gets a bit tricky if there're no test files at all because you need an extra flag and it can spawn a weird temporary file if it fails, so then tsc becomes an attractive option